perun-openapi/openapi.yml

```diff
@@ -10110,6 +10133,28 @@ paths:
         - MembersManager
       operationId: removeSponsor
       summary: Removes sponsor of existing member.
+      parameters:
+        - $ref: '#/components/parameters/memberId'
+        - name: sponsorIds
+          description: "list of ids of sponsors"
+          schema:
+            type: array
+            items:
+              type: integer
+          in: query
+          required: true
```

Comment:
These parameters should be lower in the method removeSponsor**s** instead of in the old, not bulk method removeSponsor.